frontend/src/components/Account/editPassword.module.css

```diff
@@ -0,0 +1,73 @@
+.profileForm{
+    width: 592px;
+    height: 421px;
+    display: flex;
+    background-color: #FFFFFF;
+    border-radius: 6px; 
+    flex-direction: column; 
+    border: 1px solid #D7D7D7;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+    box-sizing: border-box; 
+    align-items: center;
+    gap: 1.5em;
+    margin-bottom: 3em; 
+} 
+
+.profileFieldTitles {
+    font-size: 12px;
+} 
+
+.profileFields {
+    border-radius: 6px;
+    border: 1px solid #D7D7D7;
+    width: 551.71px; 
+    height: 51.78px;
+    text-align: center;
+    font-size: 12px;
+}
+
+.profileButton{
+    border: 1px solid #EF476F;
+    height: 16px;
+    width: 70px;
+    left: 122px;
+    top: 295px;
+    border-radius: 6px;
+}
+
+.buttonContainer{
+    flex-direction: row;
+}
+
+.editButton, .cancelButton, .saveButton {
+    width: 216.96px; 
+    height: 43.6px; 
+    left: 122px;
+    top: 295px;
+    border-radius: 6px;
+}
+
+.editButton, .saveButton {
+    border: 1px solid #EF476F;
+    background-color: #EF476F;
+    color: white;
+}
+
+.cancelButton{
+    border: 1px solid #EF476F;
+    background-color: #FFFFFF; 
+    color: black; 
+    margin-right: 5em;
+}
+
+.editButton {
+    margin-left: 21em;
+}
+.buttonContainer{
+    flex-direction: row;
+}
+
+.profileNotifTitles{
```

Comment:
not a huge deal, but we don't really gain that much from saving a few chars. Might as well be explicit and call this `profileNotificationTitles`. In this case, notif, is pretty clear but just trying to build some better habits.